source/_integrations/amcrest.markdown

```diff
@@ -143,6 +143,11 @@ Newer Amcrest firmware may not work, then `rtsp` is recommended instead.
 make sure to follow the steps mentioned at [FFmpeg](/integrations/ffmpeg/)
 documentation to install the `ffmpeg`.
 
+**Note:** From 108.5 a new motion detection mechanism has been implemented for Amcrest cameras
+to improve speed response, whilst lowering network demands. Some cameras have a 
+propreitary bug in the firmware (V2.420.0009.0.R.20151106) and will return a 500 Bad Response
+error and are incompatible with Motion Detection.
```

Comment:
Small typo. Furthermore, maybe we should use the infoboxes we have for this:

<https://developers.home-assistant.io/docs/documentation_create_page#html>

```suggestion
**Note:** From 108.5 a new motion detection mechanism has been implemented for Amcrest cameras
to improve speed response, whilst lowering network demands. Some cameras have a 
proprietary bug in the firmware (V2.420.0009.0.R.20151106) and will return a 500 Bad Response
error and are incompatible with Motion Detection.
```